### PR TITLE
Changes to sent e-mail

### PR DESCRIPTION
**Pull request by pferreir**
Added official wording and mention of the date intervals.

### CODE REVIEW

File: cern_access/indico_cern_access/templates/emails/identity_data_form_email.html [hunk 1]
```diff
@@ -1,22 +1,63 @@
...
+    {%- set url = url_for_plugin('cern_access.access_identity_data', registration.locator.uuid,
+                                                                    _external=true) -%}
     <p>Dear {{ registration.full_name }},</p>
     <p>
-        {%- set event = registration.registration_form.event -%}
-        {%- set url = url_for_plugin('cern_access.access_identity_data', registration.locator.uuid,
-                                                                         _external=true) -%}
-        The organizers of the event "{{ event.title }}" have requested access to the CERN premises for you.<br>
-        To grant you this access, we need you to fill in some security information in <a href="{{ url }}">this form</a>.
+        The organisers of the event "{{ event.title }}" have requested access to the CERN premises for you
```
Comment (ThiefMaster) [inline cern_access/indico_cern_access/templates/emails/identity_data_form_email.html]: we use "organizer" everywhere else (and both is valid in british english)

File: cern_access/indico_cern_access/templates/emails/identity_data_form_email.html [hunk 2]
```diff
@@ -1,22 +1,63 @@
...
+        In accordance with CERN’s data privacy policy, this data will be deleted from our Access database as soon as it is
+        not required anymore.
+    </p>
+    <p>
+        Upon reception of this data, you will receive an automatic email, with your access card in PDF format and an access
+        code.
     </p>
     <p>
-        Best regards,<br>
+        No access on the CERN site will be granted without complying with this formality.
```
Comment (ThiefMaster) [inline cern_access/indico_cern_access/templates/emails/identity_data_form_email.html]: access *to*?

File: cern_access/indico_cern_access/templates/emails/identity_data_form_email.html [hunk 3]
```diff
@@ -1,22 +1,63 @@
...
+                                                                    _external=true) -%}
     <p>Dear {{ registration.full_name }},</p>
     <p>
-        {%- set event = registration.registration_form.event -%}
-        {%- set url = url_for_plugin('cern_access.access_identity_data', registration.locator.uuid,
-                                                                         _external=true) -%}
-        The organizers of the event "{{ event.title }}" have requested access to the CERN premises for you.<br>
-        To grant you this access, we need you to fill in some security information in <a href="{{ url }}">this form</a>.
+        The organisers of the event "{{ event.title }}" have requested access to the CERN premises for you
+        for the period: {{ render_access_dates(start_dt, end_dt, locale='en_GB', separator='to') }}.<br/>
```
Comment (ThiefMaster) [inline cern_access/indico_cern_access/templates/emails/identity_data_form_email.html]: no need for the xml slashes

File: cern_access/indico_cern_access/templates/emails/ticket_email.html [hunk 4]
```diff
@@ -8,8 +11,11 @@
     {% set event = registration.registration_form.event %}
     <p>Dear {{ registration.full_name }},</p>
     <p>
-        You have been granted access to the CERN site during the event.
+        You have been granted access to the CERN site for the period:
+        {{ render_access_dates(start_dt, end_dt) }}.
         You can find your ticket attached to this email.
+        This ticket should be printed and presented at the CERN gate in
+        order to enter the Organisation site.
```
Comment (ThiefMaster) [inline cern_access/indico_cern_access/templates/emails/ticket_email.html]: just "to enter the site"? sounds a bit legalese with the "Organization" there

File: cern_access/indico_cern_access/util.py [hunk 5]
```diff
@@ -194,7 +197,8 @@ def withdraw_event_access_request(req):
 
 def get_random_reservation_code():
     """Generate random reservation code for data required by ADaMS API."""
-    return 'I' + ''.join(random.sample(string.ascii_uppercase.replace('O', '') + string.digits, 6))
+    charset = string.ascii_uppercase.replace('O', '').replace('0', '') + string.digits
```
Comment (ThiefMaster) [inline cern_access/indico_cern_access/util.py]: Should we get rid of I/1 as well?

Comment (pferreir) [reply cern_access/indico_cern_access/util.py]: I don't think those are as misleading. Most fonts handle them well.

File: cern_access/indico_cern_access/placeholders.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core import signals
```
Comment (ThiefMaster) [inline cern_access/indico_cern_access/placeholders.py]: unused import

File: cern_access/indico_cern_access/util.py (hunk 5 above)
Comment (ThiefMaster) [reply cern_access/indico_cern_access/util.py]: `0` is in `string.digits`, not `string.ascii_uppercase`

Comment (ThiefMaster) [reply cern_access/indico_cern_access/util.py]: maybe the whole thing is more readable if we just put a string `'ABC...Z123456789'` there?

**Pull request merged**